pytypes/typechecker.pyi

```diff
@@ -0,0 +1,48 @@
+# Generated using mypy stubgen, then editing to add types for some definitions.
+
+import __builtin__ as builtins
+from . import type_util, util
+from .exceptions import InputTypeError, OverrideError, ReturnTypeError
+from .type_util import TypeAgent, deep_type, get_orig_class, has_type_hints, is_Tuple, is_builtin_type, type_str
+from .util import getargspecs
+from types import ModuleType
+from typing import Any, Optional, Callable, TypeVar, Union
+from warnings import warn
+
```

Comment:
The import section seems to be mostly unneeded for the stub. I guess mypy stubgen simply copied the whole import section of the original module. Is that required? Otherwise I would prefer to have it cleaned up. I think only the imports from ``types`` and ``typing`` should be required.

Reply:
Yep, another place where I was erring on the side of sticking with the generated output and not getting too clever, but removed that one, and also if we're going to be going this far with curating, I'll go all the way and remove everything internal and everything that doesn't look like it's meant to be imported by other files. That also takes care of the two TODOs.

Caveat: `_pytypes___import` *is* used referenced in one place in util.py, but I don't think type checkers care about that reference.